dashboard/app/models/levels/aichat.rb

```diff
@@ -33,6 +33,7 @@ class Aichat < Level
     aichat_settings
     starter_assets
     submittable
+    disable_edit_run_for_submission
```

Comment:
Very much open to alternate/less wordy naming suggestions 🙂

Reply:
I thought about it...I got nothing 😅